src/chart_types/xy_chart/renderer/canvas/xy_chart.tsx

```diff
@@ -84,6 +84,7 @@ interface ReactiveChartDispatchProps {
 interface ReactiveChartOwnProps {
   forwardStageRef: RefObject<HTMLCanvasElement>;
 }
+const CLIPPING_MARGINS = 0.5;
```

Comment:
Can/should it be shared with `CHROME_PINCH_BUG_EPSILON` or be one defined by the other?

Reply:
... on a 2nd thought, it feels like, two constants altogether would suffice:
- `CHROME_PINCH_BUG_EPSILON` (for testing, as you already do
- `CHROME_PINCH_BUG_WORKAROUND_OFFSET` (for offsetting)
and maybe not worth making one depend on the other, though it'd be nice if they were right next to one another, with the comment about the chromium bug link and short explanation atop of it. Still in the nit category, just lets us centralize the workaround and its eventual hopeful reversal of this bad bug

Reply:
@monfera I think the idea here is to be generic to expand this later to prevent clipping of lines, points, areas, etc outside of the clipped range. In this case it should be the value of `SMALL_PIXEL` so maybe something like this would suffice.

```suggestion
const CLIPPING_MARGINS = SMALL_PIXEL;
```

But for the purposes of backporting this, I think this can wait for another time.